CITATION.cff

```diff
@@ -5,87 +5,116 @@
- - given-names: David 
-   family-names: Schneller
-   orcid: 'https://orcid.org/0000-0002-2145-6133'
- - given-names: Vikas
-   family-names: Kurapati
-   orcid: 'https://orcid.org/0009-0001-7682-508X'
- - given-names: Adrian 
-   family-names: Knoll
- - given-names: Duo 
-   family-names: Li
- - given-names: Ravil 
-   family-names: Dorozhinskii
- - given-names: Alexander 
-   family-names: Heinecke
- - given-names: Stephanie 
-   family-names: Wollherr
- - given-names: Marius 
-   family-names: Bohn
- - given-names: Nico 
-   family-names: Schliwa
- - given-names: Gilbert 
-   family-names: Brietzke
- - given-names: Taufiq 
-   family-names: Taufiqurrahman
- - given-names: Sebastian 
-   family-names: Anger
- - given-names: Sebastian 
-   family-names: Rettenberger
- - given-names: Frédéric 
-   family-names: Simonis
- - given-names: Alice 
-   family-names: Gabriel
- - given-names: Viktoria 
-   family-names: Pauw
- - given-names: Alexander 
-   family-names: Breuer
- - given-names: Fabian 
-   family-names: Kutschera
- - given-names: Kadek 
-   family-names: Hendrawan Palgunadi
- - given-names: Leonhard 
-   family-names: Rannabauer
- - given-names: Lukas 
-   family-names: van de Wiel
- - given-names: Bo 
-   family-names: Li
- - given-names: Calum 
-   family-names: Chamberlain
- - given-names: Jeena 
-   family-names: Yun
- - given-names: John 
-   family-names: Rekoske
- - given-names: Yonatan 
-   family-names: G
- - given-names: Michael 
-   family-names: Bader
-
 repository-code: 'https://github.com/SeisSol/SeisSol'
 url: 'https://seissol.org'
 abstract: >-
-  SeisSol is a scientific software for the numerical
-  simulation of seismic wave phenomena and earthquake
-  dynamics. It is based on the discontinuous Galerkin method
-  combined with ADER time discretization.
+  SeisSol is an open-source software for the simulation
+  of seismic wave phenomena and earthquake dynamics.
+  Seismic wave propagation is implemented for elastic,
+  viscoelastic and poroelastic media, based on high-order
+  discontinuous Galerkin discretization and ADER time
+  stepping on unstructured adaptive tetrahedral meshes.
+  SeisSol is optimized for large supercomputing platforms,
+  using MPI+OpenMP for parallelization and MPI+CUDA/HIP/SYCL
+  for GPU architectures.
 license: BSD-3-Clause
+keywords:
+  - earthquake simulation
+  - high performance computing
+  - discontinuous Galerkin
+
+contact:
+  - affiliation: "University of California, San Diego"
+    email: algabriel@ucsd.edu
+    family-names: Gabriel
+    given-names: Alice-Agnes
+    orcid: 'https://orcid.org/0000-0003-0112-8412'
+  - affiliation: "Technical University of Munich"
+    email: bader@cit.tum.de
+    family-names: Bader
+    given-names: Michael
+    orcid: 'https://orcid.org/0009-0000-4334-1938'
+
+# use as surrogate for "acknowledgements"; as the
+# citation format doesn't seem to provide a field for it yet
+message: >-
+  We especially acknowledge the work of Martin Käser
```

Comment:
Language edits: 
"We gratefully acknowledge Martin Käser and Michael Dumbser, originators and developers of the first version of SeisSol, as well as early contributors Cristobal Castro, Verena Hermann, and Josep de la Puente. 
We also extend our sincere thanks to all who have contributed to SeisSol's documentation, benchmarking, postprocessing, workflows, testing, and bug fixes, including Gilbert Brietzke, Duo Li, Fabian Kutschera, Kadek Hendrawan Palgunadi, Victoria Pauw, John Rekoske, Taufiqurrahman, and many others."